src/Cohesity.Powershell/Scripts/VirtualIP/Get-CohesityVirtualIP.ps1

```diff
@@ -0,0 +1,67 @@
+class VirtualIP {
+    [string]$Ip
+    [string]$VlanName
+    [string]$Hostname
+    [string]$Gateway
+    [string]$Subnet
+
+    VirtualIP($ip,$vlanName,$hostname,$gateway,$subnet){
+        $this.Ip = $ip
+        $this.VlanName = $vlanName
+        $this.Hostname = $hostname
+        $this.Gateway = $gateway
+        $this.Subnet = $subnet
+    }
+}
+function Get-CohesityVirtualIP {
+    <#
+        .SYNOPSIS
+        Get virtual IP(s).
+        .DESCRIPTION
+        The Get-CohesityVirtualIP function is used to get virtual IP(s).
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Get-CohesityVirtualIP -VlanName "intf_group2.0"
+    #>
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]
+        [string]$VlanName
+    )
```

Comment:
Can you add a new line after the parameter declaration? which will be followed in all the commandlets

Reply:
Fixed